zubhub_frontend/zubhub/src/views/create_team/CreateTseam.jsx

```diff
@@ -0,0 +1,453 @@
+import {
+  Box,
+  CircularProgress,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Grid,
+  Link,
+  Typography,
+  makeStyles,
+  useMediaQuery,
+} from '@material-ui/core';
+import {
+  ArrowBackIosRounded,
+  ArrowForwardIosRounded,
+  CloseOutlined,
+  CloudDoneOutlined,
+  Person,
+} from '@material-ui/icons';
+import DoneRounded from '@material-ui/icons/DoneRounded';
+import KeyboardBackspaceRoundedIcon from '@material-ui/icons/KeyboardBackspaceRounded';
+import clsx from 'clsx';
+import { useFormik } from 'formik';
+import PropTypes from 'prop-types';
+import { useEffect, useRef, useState, lazy } from 'react';
+import { AiOutlineExclamationCircle } from 'react-icons/ai';
+import { connect } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import StepWizard from 'react-step-wizard';
+import { toast } from 'react-toastify';
+import { colors } from '../../assets/js/colors';
+import styles from '../../assets/js/styles';
+import Modal from '../../components/modals/Modal';
+import CustomButton from '../../components/button/Button';
+import { useDomElementHeight } from '../../hooks/userDomElementHeight.hook';
+import * as AuthActions from '../../store/actions/authActions';
+import * as ProjectActions from '../../store/actions/projectActions';
+import * as UserActions from '../../store/actions/userActions';
+import { createProjectStyle } from './createTeam.style';
+import * as script from './script';
+import Step1 from './step1/Step1';
+import Step2 from './step2/Step2';
+import Step3 from './step3/Step3';
+// import { TEAM_ENABLED } from '../../utils.js';
+
+// const PreviewProject = lazy(() => import('../../components/previewProject/PreviewProject'));
+
+const DRAFT_STATUSES = { saved: 'SAVED', saving: 'SAVING', idle: 'IDLE' };
+const steps = ['Team Details', 'Add Team Members', 'Select Team Project'];
+
+function CreateTseam(props) {
+  const [completedSteps, setcompletedSteps] = useState([]);
+  const { height } = useDomElementHeight('navbar-root');
+  const isSmallScreen = useMediaQuery(theme => theme.breakpoints.down('sm'));
+  const wizardRef = useRef(null);
+  const classes = makeStyles(createProjectStyle)({ height });
+  const commonClasses = makeStyles(styles)();
+  const [draftStatus, setDraftStatus] = useState(DRAFT_STATUSES.idle);
+  const [activeStep, setActiveStep] = useState(1);
+  const [state, setState] = useState({ ...JSON.parse(JSON.stringify(script.vars.default_state)) });
```

Comment:
Not a good idea to do this. Try to use clonedeep from lodash if lodash is available. If not 
```suggestion
  const [state, setState] = useState(() => ({ ...JSON.parse(JSON.stringify(script.vars.default_state)) }));
```